packages/wdio-static-server-service/package.json

```diff
@@ -30,10 +30,13 @@
     "@wdio/logger": "6.4.7",
     "express": "^4.14.0",
     "fs-extra": "^9.0.0",
-    "morgan": "^1.7.0"
+    "morgan": "^1.7.0",
+    "@types/express": "^4.17.8",
+    "@types/fs-extra": "^9.0.1",
+    "@types/morgan": "^1.9.1"
   },
   "publishConfig": {
     "access": "public"
   },
   "types": "static-server-service.d.ts"
-}
+}
```

Comment:
let's keep the existing style of EOL if possible

Reply:
+1 , if you install the `editorconfig` plugin for your IDE this should go away

Reply:
Done